ports/supercakeboy/gameinfo.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<gameList>
+  <game>
+    <path>./Super Cakeboy</path>
```

Comment:
gameinfo.xml path needs to point to the .sh

Reply:
This still needs fixing.